### ISSUE TITLE
information addition about satellite imagery

### ISSUE THREAD

**Issue by MLS2021** [cvlab-stonybrook/ZoomLDM#4]
Thank you for your excellent work. Your publication from last year, "Learned representation-guided diffusion models for large-image generation," was also outstanding.I have a question. Both of your papers involve satellite imagery. In the first paper, you mentioned using the Chesapeake Land Cover dataset to train the diffusion model. Your second paper also deals with satellite images, a topic I am very interested in.If I want to reproduce the tasks related to satellite imagery, and I have already downloaded the pre-trained weights, how should I prepare the dataset? Is this process consistent with the data preparation procedure for the pathology cell dataset?For example ，regarding "Patch extraction," should the deepzoom_tiler.py script also be used for satellite images?"I would like to get some information about satellite image processing.Thank you very much.

**Comment by AlexGraikos** [comment]
Hello, and thanks for your interest in our work.

I have created an example gist for the multi-scale NAIP patch extraction. You will need to put all the `.tif` tile files into a single directory and run the script below.

https://gist.github.com/AlexGraikos/610f432fc305a8252afca5ff784bdadf

If you downloaded the Chesapeake dataset from [here](https://lila.science/datasets/chesapeakelandcover) (or have a subset of it), just separate the NAIP images from the other data layers (buildings, labels, etc.) and run the script.

After that, you'll need to extract SSL embeddings for every patch -- we used DINOv2 as the encoder to get the embeddings for every patch. Let me know if you'd like some guidance on running that as well.

**Comment by MLS2021** [comment]
I will conduct the experiment as soon as possible. I'm no longer in the laboratory at present. My ultimate goal is to generate satellite images.

**Comment by srikarym** [comment]
We use a pretrained Dino-v2 Large ([dinov2_vitl14_reg](https://github.com/facebookresearch/dinov2?tab=readme-ov-file#pretrained-backbones-via-pytorch-hub)) and apply it on satellite images without any finetuning. We'll upload a notebook for generating samples from our satellite diffusion model soon.

**Comment by MLS2021** [comment]
Ok. Thank you both for your answers. I am very interested in the notebook for generating samples from your satellite diffusion model and look forward to your subsequent work

**Comment by srikarym** [comment]
See [this notebook](https://github.com/cvlab-stonybrook/ZoomLDM/blob/main/notebooks/sample_patches_naip.ipynb) to sample from the satellite model.

**Comment by MLS2021** [comment]
> Hello, and thanks for your interest in our work.
> 
> I have created an example gist for the multi-scale NAIP patch extraction. You will need to put all the `.tif` tile files into a single directory and run the script below.
> 
> https://gist.github.com/AlexGraikos/610f432fc305a8252afca5ff784bdadf
> 
> If you downloaded the Chesapeake dataset from [here](https://lila.science/datasets/chesapeakelandcover) (or have a subset of it), just separate the NAIP images from the other data layers (buildings, labels, etc.) and run the script.
> 
> After that, you'll need to extract SSL embeddings for every patch -- we used DINOv2 as the encoder to get the embeddings for every patch. Let me know if you'd like some guidance on running that as well.

Hello,
First of all, thank you very much for your excellent work and the recently updated documentation. Following the new guides, I have successfully reproduced the joint multiscale and sample patches tasks for satellite imagery.
I have a follow-up question regarding the feature extraction stage: Are there any plans to release the training code for the part that involves using DINOv2 as the encoder to get the embeddings for every patch? The demo data you provided already includes the embeddings for each image patch.I'm very interested in implementing this specific step but am currently a bit unclear on the exact procedure. Any guidance you could provide on this would be greatly appreciated.
Thanks again!

**Comment by MLS2021** [comment]
Thank you for your reply. At present, I have finished the progress of data prepared, and am
 training the model, so I closed the issue.Once more thank both of your help. If it is worked well, I can add the information about satellite imagery.
